### ISSUE TITLE
Form validation demo: First reset button looks like an empty space

### ISSUE THREAD

**Issue by EricDunsworth** [wet-boew/wet-boew#8899]
On the [form validation plugin's demo page](https://wet-boew.github.io/v4.0-ci/demos/formvalid/formvalid-en.html), the first submit button is currently followed by a reset button that looks like an empty space. The empty space is a linked area and can be focused onto.

#8699 revised that button several months ago to make it stand out less. Those changes involved backporting a few Bootstrap 4 classes into WET. #8738 subsequently revised the value of the ``pt-4`` class' ``padding-top`` property. That seems to have broken the reset button.

**#8699 used this:**
```css
.pt-4,
.py-4 {
	padding-top: 1.5rem !important;
}
```

**#8738 changed it to this:**
```css
.pt-4,
.py-4 {
	padding-top: 30px !important;
}
```

@duboisp @GormFrank Any chance this class could be revised again to fix the reset button?

**Comment by thekodester** [comment]
Maybe the reset button should be changed to follow https://wet-boew.github.io/v4.0-ci/demos/feedback/feedback-en.html ?

**Comment by EricDunsworth** [comment]
@thekodester Good catch :)! Hadn't initially realized that #8699 revised several reset buttons. Looks like it wasn't just the form validation demo's reset button that was using the ``pt-4`` class...

Once #8898 gets merged, I'll try to go over all reset buttons and change them to match what's being done on the feedback demo (if it makes sense in context).
